### PR TITLE
Support cuSPARSELt v0.1.0 and cuSPARSELt choice

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

**Comment by kmaehashi** [comment]
@takagi Is this ready for review?

**Comment by takagi** [comment]
No, I also need to change where calls library choices.

**Comment by takagi** [comment]
pfnCI, test this please.

**Comment by takagi** [comment]
Let me trigger CI.

File: docker.py [hunk 1]
```diff
@@ -959,6 +973,10 @@ def get_scipy_choices():
     cusparselt_ver='0.0.1',
 )
 
+codes['cusparselt-0.1.0'] = cusparselt_base.format(
```
Comment (kmaehashi) [inline docker.py:976]: Could you check Jenkins failure and flake8 error as well?
I think the key names must be `cusparselt0.0.1-cudaXXX`? Same for `codes['cusparselt-0.0.1']`

**Comment by takagi** [comment]
pfnCI, test this please.

**Pull request merged**